index.js

```diff
@@ -1,4 +1,6 @@
 (function() {
+  var isBrowser = (typeof window !== 'undefined');
+  var React = isBrowser ? window.React || require('react') : false;
```

Comment:
I'm more tempted to call this `hasWindow`, since we don't care if something's a browser, but we do care that they support a `window` and `document` global (which could be something like Node.js with a DOM library implementation without actually being a browser in the slightest). Might I suggest this instead:

```
if (typeof window === "undefined") {
  return console.error("This environment lacks 'window' support.");
}

if (typeof document === "undefined") {
  return console.error("This environment lacks 'document' support.");
}

var React = window.React || require('react');
...
```